### PR TITLE
5057 action discord bot modify guild member

### PR DESCRIPTION
**Pull request by luancazarine**
Action
  - Modify Guild Member

### CODE REVIEW

File: components/discord_bot/actions/modify-guild-member/modify-guild-member.mjs [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+          guildId,
+          notAllowedChannels: NOT_ALLOWED_CHANNELS,
+        }),
+      ],
+      optional: true,
+    },
+    comunicationDisabledUntil: {
+      type: "string",
+      label: "Comunication Disabled Until",
+      description: "When the user's [timeout](https://support.discord.com/hc/en-us/articles/4413305239191-Time-Out-FAQ) will expire and the user will be able to communicate in the guild again (up to 28 days in the future), set to null to remove timeout. Will throw a 403 error if the user has the ADMINISTRATOR permission or is the owner of the guild",
```
Comment (alysonturing) [inline components/discord_bot/actions/modify-guild-member/modify-guild-member.mjs:83]: Can you add examples of how the user could fill that field? e.g: `1 hour` or `10 days` or the way its supposed to be filled?

File: components/discord_bot/package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@pipedream/discord_bot",
-  "version": "0.4.0",
+  "version": "0.5.0",
```
Comment (alysonturing) [inline components/discord_bot/package.json:3]: Since it doesn't have major changes, it should be `0.4.1`, do you agree?

**Review by alysonturing — CHANGES_REQUESTED** [review]
Please check the minor issues pointed

File: components/discord_bot/actions/modify-guild-member/modify-guild-member.mjs [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+const { discord } = common.props;
+
+const {
+  GUILD_TEXT,
+  GUILD_NEWS,
+  ...OTHER_CHANNELS
+} = constants.CHANNEL_TYPES;
+
+const NOT_ALLOWED_CHANNELS =
+  Object.keys(OTHER_CHANNELS).map((key) => OTHER_CHANNELS[key]);
```
Comment (GTFalcao) [inline components/discord_bot/actions/modify-guild-member/modify-guild-member.mjs:14]:
```suggestion
  Object.values(OTHER_CHANNELS);
```

File: components/discord_bot/actions/modify-guild-member/modify-guild-member.mjs [hunk 4]
```diff
@@ -0,0 +1,114 @@
...
+      userId,
+      nick,
+      roles,
+      mute,
+      deaf,
+      channel_id: channelId,
+      communication_disabled_until: comunicationDisabledUntil,
+    });
+
+    $.export("$summary", `Member with Id ${userId} Successfully modified!`);
```
Comment (GTFalcao) [inline components/discord_bot/actions/modify-guild-member/modify-guild-member.mjs:111]:
```suggestion
    $.export("$summary", `Member with Id ${userId} successfully modified!`);
```

**Review by GTFalcao — COMMENTED** [review]
Hi @luancazarine , thanks for your contribution!

I've left two comments regarding one small code suggestion and one minor grammar suggestion.

**Review by GTFalcao — APPROVED** [review]
Looks good to me! 👏

**Review by alysonturing — APPROVED** [review]

**Pull request merged**